common/src/main/java/com/wynntils/models/entities/WynntilsCustomGlowEntityProperty.java

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
I don't think this belongs in models.

In fact, do you really need this class at all? Can't we just be happy by extending the Entity class with two new methods? Hm, no, of course not, you'll need a way to access these. 

How has we solved this before? `AnnotatedItemStack` is in `com.wynntils.handlers.item` but I'm honestly not sure that is the right place.

Maybe we need a new package in `mc`. Like `mc.extensions`? 

Or, you can put this file here for the time being, if you are okay with me moving it later on.

Reply:
Exactly. I knew this is in the wrong place, I tried to look at ScreenMixin, which does the same thing, and it is in the model (as sometimes it is used in custom screen impls, so you probably got that idea). So yes, I would like you to move this :). But, don't forget to change the mixin file to the correct class name (I have no reason why this even worked before...).

Reply:
Sorry, which mixin file has the wrong name?

Reply:
But yes, leave it here and I'll clean up later.